atomics/T1127/src/T1127CustomTask.csproj

```diff
@@ -0,0 +1,9 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+   
+   <UsingTask TaskName="T1127" AssemblyFile="C:\AtomicRedTeam\atomics\T1127\src\T1127.dll" />
```

Comment:
Could you use an environment variable for the location to store the output DLL? Not everyone installs/runs atomic red team from c:\AtomicRedTeam. For example, using the $(Temp) build parameter. We generally like to avoid having the test write to the Atomic red team install directory so this also supports that model.

![image](https://user-images.githubusercontent.com/22311332/73772820-2d31bc00-473e-11ea-8be7-64794c6c87ab.png)

Reply:
Good Idea.  Let me fix that too